src/main/java/codes/biscuit/skyblockaddons/core/DungeonPlayer.java

```diff
@@ -16,17 +16,70 @@
 
     @Getter @Setter
     public static class Milestone {
-        private DungeonClass dungeonClass;
-        private String level;
-        private String value;
-
         public static Milestone zero(DungeonClass dungeonClass) {
             Milestone milestone = new Milestone();
             milestone.setDungeonClass(dungeonClass);
             milestone.setLevel("⓿");
             milestone.setValue("0");
             return milestone;
         }
+
+        private DungeonClass dungeonClass;
```

Comment:
Please move these inner classes to the bottom of the file.

Reply:
Done! Applied the changes in https://github.com/BiscuitDevelopment/SkyblockAddons/blob/0452a653d5f246a67e98aef61bbaaa57eff79e97/src/main/java/codes/biscuit/skyblockaddons/core/DungeonPlayer.java